src/styles/blocks/header.scss

```diff
@@ -0,0 +1,70 @@
+.nav {
+  display: flex;
+  flex-direction: row;
```

Comment:
```suggestion
```
default value

Reply:
Not fixed
<img width="893" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/ce7ce7b3-896b-45e0-ad70-975367b16705">